problems/[230227] 108. Convert Sorted Array to Binary Search Tree.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/convert-sorted-array-to-binary-search-tree
+ * Definition for a binary tree node.
+ * function TreeNode(val, left, right) {
+ *     this.val = (val===undefined ? 0 : val)
+ *     this.left = (left===undefined ? null : left)
+ *     this.right = (right===undefined ? null : right)
+ * }
+ */
+/**
+ * @param {number[]} nums
+ * @return {TreeNode}
+ */
+const sortedArrayToBST = (nums) => {
+  if (!nums.length) return null;
+
+  const mid = Math.floor(nums.length / 2);
+  const root = new TreeNode(nums[mid]);
```

Comment:
아주아주 사소하지만
매 재귀마다 
`this.left = (left===undefined ? null : left)` 
`this.left = (left===undefined ? null : left)`라는 삼항연산자 연산을 하게 될 것 같아요.
```suggestion
  const root = new TreeNode(
    nums[mid],
    sortedArrayToBST(nums.slice(0, mid)),
    sortedArrayToBST(nums.slice(mid + 1))
  );
```
이렇게 고치면 불필요한 삼항연산이 없어질 것 같습니다!